events/janus_rabbitmqevh.c

```diff
@@ -278,6 +279,14 @@ int janus_rabbitmqevh_init(const char *config_path) {
 		JANUS_LOG(LOG_INFO, "RabbitMQ event handler enabled, %s:%d (%s) exch: (%s) exchange_type:%s\n", rmqhost, rmqport, route_key, exchange,exchange_type);
 	}
 
+	item = janus_config_get(config, config_general, janus_config_type_item, "durable");
+	if(!item || !item->value || !janus_is_true(item->value)) {
+		JANUS_LOG(LOG_INFO, "RabbitMQEventHandler: Using non durable exchange & queue\n");
+	} else {
+		JANUS_LOG(LOG_INFO, "RabbitMQEventHandler: Using durable exchange & queue\n");
+		durable = TRUE;
+	}
```

Comment:
Can you move this block before the "exchange" property is read? The previous log line in fact is a summary of the whole configuration, and so assumes everything has been parsed already.